src/convertToObject.js

```diff
@@ -7,6 +7,22 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const splitString = sourceString
+    .split(';')
+    .filter((string) => string.includes(':'));
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. You can utilize `map` to transform each line into an object and then use `Object.assign` to merge all objects into one.